tests/utils/content.js

```diff
@@ -0,0 +1,123 @@
+const {
+  getAgeGroupIdForActivity,
+  getAgeGroupIdFromActivityGroup,
+  isObject,
+} = require("../../utils/content");
+
+describe("Content utils", () => {
+  let activities;
+  let activityGroups;
+  let ageGroup;
+
+  beforeAll(async () => {
+    /*
+      Create content hierarchy for testing:
+      - ageGroup
+        - activityGroup3
+          - activity-3
+          - activityGroup2
+            - activity2
+            - activityGroup1
+              - activity1
+    */
+    const activity1 = await strapi.query("activity").create({
+      title: "activity",
+      content: "test",
+    });
+    const activity2 = await strapi.query("activity").create({
+      title: "activity",
```

Comment:
These title and content fields could also include numbers after them, so we can differentiate if example activity 1 and acitivity 2 switch places/end up in wrong places or so.

Reply:
I don't think that's needed, because then we would be testing Strapi's own functionality and not our own code.

And we are already matching them by the `id`, so that tells us if they are in wrong places.